src/WebJobs.Script.Abstractions/WebJobs.Script.Abstractions.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+    <PropertyGroup>
```

Comment:
The files that you moved to this assembly - are you going to change their namespaces to Microsoft.Azure.WebJobs.Script.Abstractions?

Reply:
I started changing them, but didn't end up going through to avoid issues and a lot of refactoring. Would you suggest changing the namespaces for clarity?

Reply:
I think the important thing is to have the new package types in the proper public namespace going forward. I realize that'll create churn but it'd be good to get the folder structure + naming for the new package clean.

Reply:
Fair! I updated the PR with the proper changes. Once this is merged in, I will update Core Tools to also change the namespace. I don't think we have any other dependency that would need an update.